bpf/lib/nat46.h

```diff
@@ -22,6 +22,7 @@
 #include <linux/icmp.h>
 #include <linux/icmpv6.h>
 #include "common.h"
+#include "ipv4.h"
```

Comment:
Why is this necessary?

Reply:
I've suggested this to @aanm offline this was needed to fix a compile warning since nat46.h uses `ipv4_hdrlen()` which is only available in ipv4.h, and nat46.h gets pulled in via conntrack header.

Reply:
@joestringer 
```
# command-line-arguments
In file included from bpf/lib/conntrack.h:29,
                 from bpf/main.go:15:
bpf/lib/nat46.h: In function 'ipv4_to_ipv6':
bpf/lib/nat46.h:256:6: warning: implicit declaration of function 'ipv4_hdrlen'; did you mean 'ipv6_hdrlen'? [-Wimplicit-function-declaration]
  if (ipv4_hdrlen(ip4) != sizeof(v4))
      ^~~~~~~~~~~
      ipv6_hdrlen
```